aspnetcore/fundamentals/routing.md

```diff
@@ -343,7 +336,7 @@ The details of how precedence works are coupled to how route templates are defin
 * A segment with literal text is considered more specific than a parameter segment.
 * A parameter segment with a constraint is considered more specific than one without.
 * A complex segment is considered as specific as a parameter segment with a constraint.
-* Catch-all parameters are the least specific. See **catch-all** in the [Route template reference](#rtr) for important information on catch-all routes.
+* Catch-all parameters are the least specific. See **catch-all** in the [Route templates](#rtr) section for important information on catch-all routes.
 
 See the [source code on GitHub](https://github.com/dotnet/aspnetcore/blob/main/src/Http/Routing/src/Template/RoutePrecedence.cs#L189) for a reference of exact values.
```

Comment:
@Rick-Anderson Do you know where this was supposed to point to? Things have moved since this link as added and I'm not sure what it should change to. Alternatively, we could drop it altogether...

Reply:
I'd vote to drop it - the code doesn't have any more details than what is listed here.